corelib/src/cheatcodes.cairo

```diff
@@ -118,3 +118,27 @@ fn stop_warp(contract_address: ContractAddress) {
     cheatcode::<'stop_warp'>(array![contract_address_felt].span());
 }
 
+fn start_mock_call(contract_address: ContractAddress, fn_name: felt252, ret_data: Array::<felt252>) {
```

Comment:
I think we could use nicer sounding names in the user facing interface e.g. `function_name` instead of `fn_name`